<launch>
  <!-- YOLOv3节点 -->
  <include file="$(find darknet_ros)/launch/yolo_v3.launch"/>
  <!-- 摄像头节点 -->
  <include file="$(find usb_cam_opencv)/launch/usb_cam_opencv.launch"/> 
  <!-- 雷达节点 -->
  <include file="$(find rplidar_ros)/launch/rplidar.launch"/> 
  <!-- 雷达分割节点 -->
  <include file="$(find laser_segment)/launch/laser_segment.launch"/> 
  <!-- 分割反投影节点-->
  <node name="image_segment_node" pkg="image_segment" type="image_segment_node"  output="screen">
	  <param name="begin_ray_id_" type="int" value="150" />
	  <param name="topic_laser_seg_"  type="string" value="insight_laser_seg" />
	  <param name="topic_detect_img_" type="string"  value="cropped_detection_Image" />
  </node>
</launch>

